src/App.js

```diff
@@ -10,13 +10,69 @@ import Menu from "./pages/Menu/Menu.js";
 
 // Test world image
 import img from "./images/Earth.png"
+import Crossword from './components/Crossword/Crossword';
 
 function App() {
   const messages = [
     "Hello, This is Captain Cybot! Oh no! Evil malware has taken over the four worlds in our galaxy and each world has been infected with all sorts of malware like viruses, worms, trojans and ransomware. Join Captain Cybot on his adventure to defeat malware and free the galaxy once & for all. Are you ready to join Captain Cybot on his adventure?",
     "If you would like to see more awesome stuff, check out the other writeups at codeworkshop.dev!",
     "Remember to wash your hands!"
   ]
+  const myData = {
```

Comment:
Should be pulled from BE

Reply:
This is here temporarily for the sake of screenshots for the progress report. We will change it to communicate with the BE when it is ready for it.